cms/cms_toolbars.py

```diff
@@ -127,18 +127,18 @@ def add_admin_menu(self):
 
             if len(sites_queryset) > 1:
                 sites_menu = self._admin_menu.get_or_create_menu('sites', _('Sites'))
-                sites_menu.add_sideframe_item(_('Admin Sites'), url=admin_reverse('sites_site_changelist'))
+                sites_menu.add_modal_item(_('Admin Sites'), url=admin_reverse('sites_site_changelist'))
                 sites_menu.add_break(ADMIN_SITES_BREAK)
                 for site in sites_queryset:
                     sites_menu.add_link_item(site.name, url='http://%s' % site.domain,
                                              active=site.pk == self.current_site.pk)
 
             # admin
-            self._admin_menu.add_sideframe_item(_('Administration'), url=admin_reverse('index'))
+            self._admin_menu.add_modal_item(_('Administration'), url=admin_reverse('index'))
```

Comment:
@mkoistinen the admin links should actually all open in new windows, not the modal